code/modules/vending/_vending.dm

```diff
@@ -861,6 +870,12 @@
 /atom/movable/proc/post_tilt()
 	return
 
+/atom/movable/proc/can_be_vendorcrushed(atom_target)
```

Comment:
look at what's invoking it

Reply:
... Look at the proc. All it does is check if the passed atom can be crushed. There's no reason to be on the movable level.

Reply:
I'm just confused on how you would invoke it on the vending level then, I'm probably missing something

Reply:
Oh I see, the name of this proc greatly confused me, since it doesn't represent what it's actually doing. It should be something like `/proc/can_fall_on_and_crush(atom/crushing)` or similar. 
Then we can put the static typecache in the proc.